src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -87,39 +87,41 @@ public static function substr(?string $string, int $from, ?int $length = null):
 
     public static function formatTime(int|float $secs, int $precision = 1): string
     {
+        $ms = $secs * 1000;
         $secs = (int) floor($secs);
 
-        if (0 === $secs) {
-            return '< 1 sec';
+        if ($ms < 0.001) {
+            return '< 1 ms';
         }
 
         static $timeFormats = [
-            [1, '1 sec', 'secs'],
-            [60, '1 min', 'mins'],
-            [3600, '1 hr', 'hrs'],
-            [86400, '1 day', 'days'],
+            [1, '1 ms', 'ms'],
+            [1000, '1 sec', 'secs'],
+            [60000, '1 min', 'mins'],
+            [3600000, '1 hr', 'hrs'],
+            [86400000, '1 day', 'days'],
         ];
 
         $times = [];
         foreach ($timeFormats as $index => $format) {
-            $seconds = isset($timeFormats[$index + 1]) ? $secs % $timeFormats[$index + 1][0] : $secs;
+            $milliSeconds = isset($timeFormats[$index + 1]) ? $ms % $timeFormats[$index + 1][0] : $ms;
 
             if (isset($times[$index - $precision])) {
                 unset($times[$index - $precision]);
             }
 
-            if (0 === $seconds) {
+            if ($milliSeconds < 0.001) {
                 continue;
             }
 
-            $unitCount = ($seconds / $format[0]);
+            $unitCount = ($milliSeconds / $format[0]);
             $times[$index] = 1 === $unitCount ? $format[1] : $unitCount.' '.$format[2];
 
-            if ($secs === $seconds) {
+            if (abs($ms-$milliSeconds) < 0.001) {
```

Comment:
Since `$ms` and `$milliSeconds` should be int, the comparison `===` can be used instead of fuzzy numerical comparaison.